### PR TITLE
Added version 9.12.0 

### PR DESCRIPTION
**Pull request by bahugo**

### CODE REVIEW

**Comment by bahugo** [comment]
closes #1

File: .github/workflows/stable.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [9.11.0]
+        version: [9.11.0, 9.12.0]
```
Comment (jschueller) [inline .github/workflows/stable.yml]: lets drop 9.11

File: .github/workflows/stable.yml [hunk 2]
```diff
@@ -37,22 +37,19 @@ jobs:
...
     permissions:
-      contents: write 
+      contents: write
     strategy:
       matrix:
+        version: [9.11.0, 9.12.0]
         include:
           - abi: cp38
             python-version: '3.8'
-            version: 9.11.0
```
Comment (jschueller) [inline .github/workflows/stable.yml:46]: I think this is still needed (update it)

Comment (bahugo) [reply .github/workflows/stable.yml:46]: I tkink it should work https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations but i haven't tested it.

File: .github/workflows/stable.yml (hunk 1 above)
Comment (bahugo) [reply .github/workflows/stable.yml]: I've found another issue #3. It might be interesting to fix also 9.11.

Comment (bahugo) [reply .github/workflows/stable.yml]: Do you want me to open another pull request for #3 and then remove version 9.11 from this one?

Comment (jschueller) [reply .github/workflows/stable.yml]: lets do everything in one PR

**Comment by bahugo** [comment]
Hi, 
I've found the new download link for med-4.1.1.
I've added a pyproject.toml to simplify the built of the wheel.
But I'm still not able to compile medcoupling...
I've been focusing my efforts on windows.
I've reworked a bit the .bat file in order to be able de test it on my local windows machine.

Here are the failing compilations : 
- metis
- med
- medcoupling

I've tried (a lot) to figure out by my own what's the issue, i've event tried to use your legacy .bat (and just change the med download link) but it failed too.

Any help would be more than welcome 😉.

**Pull request closed without merging**